api/index.py

```
@@ -448,3 +449,41 @@ def switch_commit():
+
+    Returns:
+        JSON response with a list of commits or an error message.
+    """
+
+    # Get JSON data from the request
+    data = request.get_json()
+
+    # Validate the presence of the 'repo' parameter
+    repo = data.get('repo')
+    if repo is None:
+        return jsonify({'status': f'Missing mandatory parameter "repo"'}), 400
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)